includes/admin/services/class-rop-facebook-service.php

```diff
@@ -694,6 +703,14 @@ private function try_post( $new_post, $page_id, $token, $post_id, $posting_type
 			$this->set_api( $this->credentials['app_id'], $this->credentials['secret'] );
 		}
 
+		if ( ! class_exists( 'ROP_Pro_Facebook_Helper' ) ) {
+			include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+			if ( is_plugin_active( 'tweet-old-post-pro/tweet-old-post-pro.php' ) ) {
+				require_once ROP_PRO_PATH . 'includes/helpers/class-rop-pro-facebook-helper.php';
+			}
+		}
+
```

Comment:
Now that we have checks, we no longer need this forced load of the class.

If the PRO plugin can load this class, you don't need to force it unless there is a specific problem.